src/common/Section/styled.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+export const StyledSection = styled.section``;
+
+export const Title = styled.h2`
+  font-size: 36px;
+  line-height: 1.2;
+  font-weight: 600;
+  color: ${({ theme }) => theme.colors.Woodsmoke};
+  margin: 0 0 24px 0;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}) {
+    text-align: center;
+    font-size: 18px;
+    font-weight: 600;
+    line-height: 1.2;
+    margin: 0 0 12px 0;
+  } ;
+`;
+
+export const Container = styled.section`
+  display: grid;
```

Comment:
dodaj proszę propsa "grid" i tam przerzuć
```
 display: grid;
grid-template-columns: repeat(auto-fill, minmax(324px, 1fr));
  grid-gap: 24px;` 
```
ponieważ czasami to będzie grid a czasami nie :) w zależności od podstrony

Reply:
Super, że widzisz takie rzeczy już teraz, mi to póki co najczęściej wychodzi na późniejszych etapach, kiedy już trzeba taki grid zastosować, bądź też nie ;)

Reply:
Wiesz, nic by się nie stało. Bo właśnie gdyby ktoś potem potrzebował to zmienić to by zmienił ale jak już i tak będą zmiany i to miałem na uwadze to warto od razu zrobić :)